lib/yabeda/datadog/logger.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'singleton'
+
+module Yabeda
+  module Datadog
+    # = Perform loging
+    class Logger
+      include Singleton
+
+      def initialize
+        @logger = ::Logger.new(STDOUT)
+      end
+
+      def log_request(metric, &block)
+        begin
+          info "Sending #{metric.name} metric"
+          result = handle_result(yield)
+          info "Response on #{metric.name}: #{result}"
```

Comment:
It will be good to return what the passed block returns, now it returns `true` of last info method invocation.